director/background_tasks.go

```diff
@@ -377,9 +373,8 @@ func processUnconfiguredDevices() error {
 }
 
 func ScheduledCheckin() {
-	// var delay time.Duration
 	var scheduledPushes []types.ScheduledPush
-	err := db.DB.Unscoped().Model(&scheduledPushes).Delete(&types.ScheduledPush{}).Error
+	err := db.DB.Session(&gorm.Session{AllowGlobalUpdate: true}).Unscoped().Model(&scheduledPushes).Delete(&types.ScheduledPush{}).Error
```

Comment:
This is to allow us to delete all Unscoped (i.e. deleted) scheduled pushes **without a where clause**